docs/Review_Apps.md

```diff
@@ -106,13 +112,13 @@ In order to start a review app you will need to follow these steps:
     hokusai review_app deploy <name> <name>
     ```
 
-13) If you have made changes to your review app's yaml file, you need to update deployment:
+14) If you have made changes to your review app's yaml file, you need to update deployment:
 
```

Comment:
does the `deploy` step accept the new `-u` flag? If so we can probably note that on step 13

Reply:
It does!